minishift/openshift/f8notification.config.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: v1
+kind: Template
+metadata:
+  name: f8notification
+objects:
+- apiVersion: v1
+  kind: Secret
+  metadata:
+    name: f8notification
+  type: Opaque
+  data:
+    madrill.api: eHh4eHh4
+    service.account.id: NGM4M2NhMmQtNmRjYy00MWM5LWFjN2QtYjA2OGFkNGQxN2M1
+    service.account.secret: bm90aWZpY2F0aW9uc2VjcmV0
```

Comment:
this could be a parameter, so we never commit an API key in the repository

Reply:
I copied this template from notification repository. If someone has to try it then there should have some way using which we can share required keys with a user who wanted to try this locally

Reply:
yes, but what about the API key that is committed in this file ? Can *anyone* use it, really?

Reply:
If you are referring to `madrill.api`, I think it's not a valid key. I already described it in the documentation https://github.com/fabric8-services/fabric8-auth/pull/588/files#diff-cad2fe3b8c90b796038ba2d828cb8e4cR123

Reply:
ok, so that part of the docs should be rewritten to explain how to set it as an env var, instead.

Reply:
Addressed in d291bd56af2c0b37038c22386ec2d2b5a8bdd352